src/components/book-section.js

```diff
@@ -94,29 +94,29 @@ function BookSection() {
           ))}
         </div>
       ))}
-      <h3>ADD NEW BOOK</h3>
-      <section className="newBook">
-        <form onSubmit={handleAddBook}>
-          <input
-            placeholder="Book Title"
-            className="bookTitle"
-            value={title}
-            onChange={titleFunc}
-          />
-          <input
-            placeholder="Author"
-            className="bookAuthor"
-            value={author}
-            onChange={authorFunc}
-          />
-          <AddButton
-            item_id={nanoid()}
-            title={title}
-            author={author}
-            category="N/A"
-          />
-        </form>
-      </section>
+      <h3 style={{ color: 'rgba(0, 0, 0, 0.35)', paddingLeft: '7rem' }}>ADD NEW BOOK</h3>
```

Comment:
- [ ] Adjust this to look more like the design

Yours:
![image](https://user-images.githubusercontent.com/62705947/222691917-76a6dcb1-7750-4600-8064-19c06cec55d6.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/222691971-4b93a0a0-c806-4b83-add9-d58e23b7c1d6.png)